R/utilities.R

```diff
@@ -752,6 +711,31 @@ find_stan_file <- function(file, subdir = "") {
     }
 }
 
+#' Get a unique hash
+#'
+#' Gets a unique string on the string content but also including the current session
+#' packages + R version
+#'
+#' @importFrom utils sessionInfo
+#' @keywords internal
+get_unique_hash <- function(content) {
+    pkg_versions <- vapply(
+        sessionInfo(c("rbmi", "rstan", "Rcpp", "RcppEigen"))[["otherPkgs"]],
```

Comment:
I discovered yesterday that `Rcpp` and `RcppEigen` are not strictly required to be installed at runtime, because `stan` only depends on them as `linking-to` (similar to the issues we had with `BH`).

Reply:
Further tweaked in c334ab1a in response to feedback from @luwidmer